README.md

```diff
@@ -62,9 +74,9 @@ To start automatic rebalancing with the Cellars rebalancer, run the start comman
 CELLAR_DRY_RUN=false cargo run -- -c [your_config_file_name.toml] start
 ```
 
-### Cellar Rebalancer Subcommands
+### Steward Subcommands
 
-Below is a list of the Cellar rebalancer's subcommands:
+Below is a list of the Steward's subcommands:
```

Comment:
We'll need to add an entry to the subcommand list for the single signer test mode (and I think `cosmos-signer` as well, though I'm not sure if `start` is also starting the Cosmos signer mode).

Reply:
There's no start command. To start the single signer test mode, you'll have to run the `single-signer` command. You can also run `cosmos-signer` to start the cosmos-signer command. Sorry for the misleading info, I have upgraded the readme.